listMembersOfaJobGroups.groovy

```diff
@@ -0,0 +1,17 @@
+import nectar.plugins.rbac.groups.*
+
+def job = Jenkins.instance.items.each{it instanceof Job}.find{it.name == 'CopyG'}
+
+if(job instanceof Job){
```

Comment:
It might be better to just do:

```
def job = Jenkins.instance.items.getAllItems(Job.class).each {it ->
  if (it.name == 'CopyG') {
    ......
  }
}
```